web/views.py

```diff
@@ -110,6 +111,7 @@ def get_jspsych_response(context, is_preview=False):
         exp_data=[],
         is_preview=is_preview,
         study_type=study.study_type,
+        recording_method="jspsych",
```

Comment:
@okaycj so far we've been using `null`/`None` for the jsPsych recording method. Just double checking that we do want to intentionally switch to "jspsych". (FWIW, I don't think it matters!)

Maybe you've already covered this in the tests, but we just need to make sure that whatever logic we're using around `recording_method` still makes sense with jsPsych values. For instance, if we use "jspsych" as a `recording_method` value, then maybe we should add a new `recording_method_is_jspsych` property. That way we could fix this logic that checks the study type first, then the recording method:

```python
def get_url(
    video_key, recording_method_is_pipe, study_type_is_jspsych, set_attachment_header
):
    """
    Generate a presigned url for the video that expires in 10 minutes.
    """
    url = None
    if study_type_is_jspsych:
        # jsPsych bucket
        bucket = settings.JSPSYCH_S3_BUCKET
    else:
        # EFP
        if recording_method_is_pipe:
            # Pipe bucket
            bucket = settings.BUCKET_NAME
        else:
            # RecordRTC bucket
            bucket = settings.S3_BUCKET_NAME
```

I'm open to whatever make sense to you here.